core/services/gateway/handlers/workflow/handler.go

```diff
@@ -0,0 +1,152 @@
+package workflow
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+	"time"
+
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/api"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/config"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/network"
+)
+
+const (
+	MethodAddWorkflow = "add_workflow"
```

Comment:
We'll have a different handler for workflow spec CRUD. This one can be only for web capabilities.